README.md

```
@@ -287,7 +287,7 @@ Run the `deploy` command:
 serverless deploy
 ```
 
-More details on deploying can be found [here](https://www.serverless.com/framework/docs/providers/aws/guide/deploying).
+More details on deploying can be found [here](https://github.com/aripitek/www.serverless.com/framework/docs/providers/aws/guide/deploying).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)